gandalf.js

```diff
@@ -1,18 +1,17 @@
-/**
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?